tsdb/index/index.go

```diff
@@ -1786,11 +1795,17 @@ func (r *Reader) PostingsForLabelMatching(ctx context.Context, name string, matc
 		return EmptyPostings()
 	}
 
+	postingsCount := 0
```

Comment:
Is "Count" a good word to use for a guesstimate?
Is zero a good guess for the non-nil case?

Reply:
"Cap" suggests it can't go any higher.  How about `postingsEstimate` ?

Reply:
Applied your suggestion, PTAL.